packages/core/src/ui/UIRenderer.ts

```
@@ -0,0 +1,248 @@
+
+  /**
+   * @internal
+   */
+  @ignoreClone
+  _onEntityModify(flag: EntityModifyFlags): void {
+    switch (flag) {
+      case EntityModifyFlags.SiblingIndex:
+        this._rootCanvas && (this._rootCanvas._hierarchyDirty = true);
+        break;
+      case EntityModifyFlags.Parent:
+        const rootCanvas = UIUtils.getRootCanvasInParent(this._entity);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)